services/api/src/admin_new_intersection.py

```diff
@@ -25,7 +25,7 @@ def get_allowed_selections():
 
 
 def check_safe_input(intersection_spec):
-    special_characters = "!\"#$%&'()*+,./:;<=>?@[\\]^`{|}~"
+    special_characters = "!\"#$%'()*+,./:;<=>?@[\\]^`{|}~"
```

Comment:
I am not a big fan of just simply allowing the special character. Does the web application properly handle ensuring the character is treated with a proper escape key? '\&'

My concern is less of a potential SQL injection being malicious but potentially just causing straight up issues executing.

I can't get the whole thing running so I am just leaving this as a comment for now.